pages/api/augment-request.ts

```diff
@@ -79,7 +79,7 @@ function constructOoUiLink(
   const subDomain = Number(chainId) === 5 ? "testnet." : "";
   return `https://${subDomain}oracle.uma.xyz/request?transactionHash=${txHash}&chainId=${chainId}&oracleType=${castOracleNameForOOUi(
     oracleType
-  )}`;
+  )}&eventIndex=`;
```

Comment:
for some reason if this isnt included, we dont link correctly to the OO UI. even if its left blank like it is here we can deduce the correct request in the UI